src/olympia/devhub/tasks.py

```diff
@@ -620,49 +620,42 @@ def track_validation_stats(json_result, addons_linter=False):
 
 @task
 @set_modified_on
-def resize_icon(src, dst, size, locally=False, **kw):
+def resize_icon(src, dest_folder, sizes, **kw):
     """Resizes addon icons."""
-    log.info('[1@None] Resizing icon: %s' % dst)
+    log.info('[1@None] Resizing icon: %s' % dest_folder)
     try:
-        if isinstance(size, list):
-            for s in size:
-                resize_image(src, '%s-%s.png' % (dst, s), (s, s),
-                             remove_src=False, locally=locally)
-            if locally:
-                os.remove(src)
-            else:
-                storage.delete(src)
-        else:
-            resize_image(src, dst, (size, size), remove_src=True,
-                         locally=locally)
+        dest_file = None
+        for s in sizes:
```

Comment:
pep08 hates one-character names, let's maybe call `sizes` `target_sizes` instead and name `s` as `size`?